apps/emqx_durable_storage/src/emqx_ds_storage_skipstream_lts.erl

```diff
@@ -407,6 +458,8 @@ do_delete(CF, Batch, Static, KeyFamily, MsgKey) ->
 
 %%%%%%%% Iteration %%%%%%%%%%
 
+init_iterators(S, #stream{static_index = Static}) ->
```

Comment:
Q: how could we get a stream at this point?  This function seems to only called by `next/6`, whose input is an iterator.  `scan_stream` also crafts an iterator, so this clause looks unreachable.